guppylang/std/unsupported.py

```diff
@@ -0,0 +1,231 @@
+# Python builtins that are not supported yet:
+from guppylang import guppy
+from guppylang.std._internal.checker import UnsupportedChecker
+
+
+@guppy.custom(checker=UnsupportedChecker(), higher_order_value=False)
```

Comment:
These are now placed into their own Guppy module but they should still be part of the builtins Guppy module (so they are always implicitly imported).

Have look at top of `std/options.py` for how to get a handle to the builtins guppy module

Reply:
Ah thanks, I've added that now. There still seems to be some sort of issue with C.I.